src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -89,3 +89,25 @@ export const initialState: Portfolio = {
     ids: [],
   },
 }
+
+export enum AssetEquityType {
+  Account = 'Account',
+  Staking = 'Staking',
+  LP = 'LP',
+  Reward = 'Reward',
+}
+
+export type AssetEquity = {
```

Comment:
Perhaps `AssetEquityItem` would be more declarative to make it clear this represents a single item, not "equity" as a whole